benchmarks/java/src/main/java/com/google/protobuf/ProtoBench.java

```diff
@@ -43,18 +43,22 @@
 import com.google.protobuf.benchmarks.Benchmarks.BenchmarkDataset;
 import java.io.ByteArrayInputStream;
 import java.io.ByteArrayOutputStream;
+import java.io.EOFException;
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.io.RandomAccessFile;
+import java.nio.file.*;
```

Comment:
Expend this. Don't use wildcards in imports.
https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Done